anki/web/editor.js

```diff
@@ -69,27 +79,31 @@ document.addEventListener("keydown", (evt) => {
     }
 });
 
-EditorField.lifecycle.onMount(async (field) => {
-    const fieldElement = await field.element;
-
-    if (!fieldElement.hasAttribute("has-occlusion-style")) {
-        const style = document.createElement("style");
-        style.id = "closet-occlusion-style"
-        style.rel = "stylesheet";
-        style.textContent = occlusionCss;
-        const richTextEditable = await get(field.editingArea.editingInputs)
-            .find((input) => input.name === "rich-text")
-            .element;
-        richTextEditable.getRootNode().prepend(style);
-
-        fieldElement.setAttribute("has-occlusion-style", "");
-    }
-})
+EditorField.lifecycle.onMount((field) => {
+    (async() => {
+        const fieldElement = await field.element;
+
+        if (!fieldElement.hasAttribute("has-occlusion-style")) {
+            const style = document.createElement("style");
```

Comment:
Why was this change necessary (from async function to sync function with IIFE), did you encounter an issue here?

Reply:
Loading a note with fewer fields caused an error, because async callbacks are not supported: https://github.com/ankitects/anki/issues/1959.